rmc/utils/mpc.py

```diff
@@ -392,15 +406,23 @@ def prepare_pop_path_ht(
     if not file_exists(context_with_oe_dedup.path) or (
         transcript_col not in set(context_with_oe_dedup.ht().row)
     ):
-        create_context_with_oe(include_rescue=include_rescue, overwrite=overwrite)
+        create_context_with_oe(
+            include_rescue=include_rescue,
+            overwrite_temp=overwrite_temp,
+            overwrite_output=overwrite_output,
+        )
 
     # Get transcript annotation from deduplicated context HT resource
     context_ht = context_with_oe_dedup.ht()
+    if transcript_col not in context_ht.row:
```

Comment:
sorry if I suggested this check -- isn't this taken care of in the if statement above?

Reply:
See below comment and let me know if you disagree!